juju/client/connection.py

```diff
@@ -267,11 +273,13 @@ def https_connection(self):
         username = accounts['user']
         password = accounts.get('password')
         models = jujudata.models()[controller_name]
+        if '/' not in model_name:
+            model_name = '{}/{}'.format(username, model_name)
```

Comment:
It actually looks like this has always been the case in 2.x but we weren't using `Connection.connect_model()` directly before, so this is really more of a convenience that I thought was a bug before.